hedera-node/src/test/java/com/hedera/services/txns/contract/ContractCreateTransitionLogicTest.java

```diff
@@ -416,6 +421,19 @@ void resetMethodDelegates() {
 		verify(entityIdSource).resetProvisionalIds();
 	}
 
+	@Test
+	void prepareCodeWithConstructorArgumentsTest() {
```

Comment:
nit: We should specify that this unit is actually expecting a throw f.e `throwsErrorOnInvalidBytecode`